src/main/scala/com/codacy/client/bitbucket/v2/service/TeamServices.scala

```diff
@@ -11,6 +11,17 @@ class TeamServices(client: BitbucketClient) {
     client.executePaginated(Request(s"$BaseUrl?role=$role", classOf[Seq[Team]]))
   }
 
+  /**
+    * Gets the public information associated with a team.
+    *
+    * If the team's profile is private, location, website and created_on elements are omitted.
+    *
+    * @param username The username or the UUID of the account surrounded by curly-braces
+    */
+  def getTeam(username: String): RequestResponse[Team] = {
```

Comment:
Won't we have problems with encoding if this is the the UUID?

Reply:
Would we not do that on the caller of this?

Reply:
I hope that we do it here. It is a public method that says "The username or the UUID of the account surrounded by curly-braces", so I will call this with a normal "{1s123-1231x-231}" and expect it to be handled

Reply:
It is done :+1:

Reply:
During the review of @prendi PR, I see that we pass the ownerInfo: OwnerInfo, instead of username, example (getRepositories), can you check if it applies here? Maybe it should be ownerInfo, so it can be more clear that it can be either username / UUID

Reply:
I think we should keep with the name of `username` as that is what is stipulated on the Bitbucket documentation and we should remain consistent with that. Changing it to a bespoke value only we understand seems messy. 

https://developer.atlassian.com/bitbucket/api/2/reference/resource/teams/%7Busername%7D